wpeverest/user-registration
# UR-1679 Feature - WPBakery compatibility with user registration widgets.

### All Submissions:






### Changes proposed in this Pull Request:

Introducing WPBakery Compatibility by creating user registration widgets like Registration, Login, My Account, Edit Profile, Edit Password. 

### How to test the changes in this Pull Request:

1. Edit WordPress Page with WPBakery Editor. 
2. Click on + icon and you can see User Registration category with all widgets.
3. Verify all widgets are working properly or not.

### Types of changes:




### Other information:




### Changelog entry

> Feature - WPBakery compatibility with user registration widgets.
